### PR TITLE
Remove drivers dropdown in the settings tab

### PR DESCRIPTION
**Pull request by kilbith**
Already discussed plenty of times and several devs are in agreement that it's dumb and counterproductive. @sfan5 @RealBadAngel @est31 

Also remove a duplicate colored box.

### CODE REVIEW

File: builtin/mainmenu/tab_settings.lua [hunk 1]
```diff
@@ -247,7 +218,6 @@ local function formspec(tabview, name, tabdata)
 		"dropdown[0.25,3.2;3.3;dd_leaves_style;" .. leaves_style[1][1] .. ";"
 				.. getLeavesStyleSettingIndex() .. "]" ..
 		"box[3.75,0;3.75,3.45;#999999]" ..
-		"box[3.75,0;3.75,4.7;#999999]" ..
```
Comment (est31) [inline builtin/mainmenu/tab_settings.lua]: The box is not entirely duplicate, its higher than the other one. Perhaps remove the other, smaller one?

**Comment by est31** [comment]
See comments, otherwise :+1:

File: builtin/mainmenu/tab_settings.lua (hunk 1 above)
Comment (kilbith) [inline builtin/mainmenu/tab_settings.lua:250]: Those boxes overlay mostly at each other. I can't keep the larger one if the dropdown is removed (it'd have a white unfilled space, that's weird and has never been there).

Comment (est31) [inline builtin/mainmenu/tab_settings.lua]: OK then

**Comment by RealBadAngel** [comment]
+1

**Comment by nerzhul** [comment]
Merged: a8c5841140c23f246adbc58f7c73195c01253697

**Pull request closed without merging**